doc/projects.md

```diff
@@ -33,13 +33,13 @@ Add your project/integration to this file and submit a pull request.
 1. [Protractor-axe-report Plugin](https://github.com/E1Edatatracker/protractor-axe-report-plugin)
 1. [Cypress-axe](https://github.com/avanslaars/cypress-axe)
 1. [Jest-axe](https://github.com/nickcolley/jest-axe)
-1. [aXe audit runner for CrawlKit](https://github.com/crawlkit/runner-axe)
-1. [Selenium IDE aXe Extension](https://github.com/bkardell/selenium-ide-axe)
+1. [axe audit runner for CrawlKit](https://github.com/crawlkit/runner-axe)
```

Comment:
Capitalize? or not since it's the name of someone elses project?

Reply:
I didn't capitalize here since neither "audit" nor "runner" were capitalized, but I'm fine with whatever. The purpose of this PR is just to remove the capitalized X. :woman_shrugging: